config/ldap.yml

```diff
@@ -0,0 +1,31 @@
+## Authorizations
```

Comment:
This file should not be included in the repository. Add it to .gitignore, or set a generic content, something not specific for us.